sampleapi/api_ok.go

```diff
@@ -1,10 +1,32 @@
 package sampleapi
 
+import (
```

Comment:
func main() {
	var wg sync.WaitGroup

	for i := 0; i < 2; i++ {
		wg.Add(1)

		go func(i int) {
			defer wg.Done()

			fmt.Println(i)
		}(i)
	}

	wg.Wait()
}

// This code appears to be correct. It is setting up two goroutines and waiting for them to finish before exiting the main function. The variables are correctly declared with the necessary scopes, and the WaitGroup is used correctly. There are no obvious bugs or issues with the code.